src/Files/Strings/en-US/Resources.resw

```diff
@@ -2752,6 +2749,114 @@ We use App Center to keep track of app usage, find bugs, and fix crashes. All in
+  </data>
+  <data name="SearchSize_NoLimit" xml:space="preserve">
+    <value>No limit</value>
+  </data>
+  <data name="And" xml:space="preserve">
+    <value>And</value>
+  </data>
+  <data name="SearchAndFilterCollection_Description" xml:space="preserve">
+    <value>Finds items that meet all the conditions in the list.</value>
+  </data>
+  <data name="Or" xml:space="preserve">
+    <value>Or</value>
+  </data>
+  <data name="SearchOrFilterCollection_Description" xml:space="preserve">
+    <value>Finds items that meet at least one condition in the list.</value>
+  </data>
+  <data name="Not" xml:space="preserve">
+    <value>Not</value>
+  </data>
+  <data name="SearchNotFilterCollection_Description" xml:space="preserve">
+    <value>Finds items that do not meet any condition in the list.</value>
+  </data>
+  <data name="Created" xml:space="preserve">
+    <value>Created</value>
+  </data>
+  <data name="SearchCreatedFilter_Description" xml:space="preserve">
+    <value>Date of creation</value>
+  </data>
+  <data name="Modified" xml:space="preserve">
+    <value>Modified</value>
+  </data>
+  <data name="SearchModifiedFilter_Description" xml:space="preserve">
+    <value>Date of last modification</value>
+  </data>
+  <data name="Accessed" xml:space="preserve">
+    <value>Accessed</value>
+  </data>
+  <data name="SearchAccessedFilter_Description" xml:space="preserve">
+    <value>Date of last access</value>
+  </data>
+  <data name="Size" xml:space="preserve">
+    <value>Size</value>
+  </data>
+  <data name="SearchSizeFilter_Description" xml:space="preserve">
+    <value>Size of the item</value>
+  </data>
+  <data name="SearchGroupHeader_ItemSuffixe" xml:space="preserve">
+    <value>({0} item)</value>
+  </data>
+  <data name="SearchGroupHeader_ItemsSuffixe" xml:space="preserve">
+    <value>({0} items)</value>
+  </data>
+  <data name="SearchOptions" xml:space="preserve">
+    <value>Search Options</value>
+  </data>
+  <data name="File" xml:space="preserve">
+    <value>File</value>
+  </data>
+  <data name="Group" xml:space="preserve">
+    <value>Group</value>
+  </data>
+  <data name="SearchPickerLocation_Subfolders.Content" xml:space="preserve">
+    <value>Include Subfolders</value>
+  </data>
+  <data name="SearchPickerGroup_NoSelected.Text" xml:space="preserve">
+    <value>No filter selected.</value>
+  </data>
+  <data name="Add" xml:space="preserve">
+    <value>Add</value>
+  </data>
+  <data name="Clear" xml:space="preserve">
+    <value>Clear</value>
+  </data>
+  <data name="SearchPickerDateRange_NoSelected.Text" xml:space="preserve">
+    <value>No date range selected.</value>
+  </data>
+  <data name="SearchPickerDateRange_Choose.Text" xml:space="preserve">
+    <value>Or choose a date range</value>
+  </data>
+  <data name="SearchPickerSizeRange_NoSelected.Text" xml:space="preserve">
+    <value>No size range selected.</value>
+  </data>
+  <data name="SearchPickerSizeRange_Choose.Text" xml:space="preserve">
```

Comment:
Can you remove the type specific extensions so strings can be reused?

Reply:
Ok, I'll do this for all the texts in this pr.

Reply:
@yaichenbaum It is done.